FlavorTrack/Core/Favorite List/FavoriteListVC.swift

```diff
@@ -56,35 +55,40 @@ extension FavoriteListVC: UITableViewDelegate, UITableViewDataSource {
 	func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
 		allFavorites.count
 	}
-	
+
 	func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		// swiftlint:disable force_cast
 		let cell = tableView.dequeueReusableCell(withIdentifier: FavoriteListCell.reuseIdentifier,
```

Comment:
`guard let cell = tableView.dequeueReusableCell(withIdentifier: FavoriteListCell.reuseIdentifier,
                                                       for: indexPath) as? FavoriteListCell else {
            fatalError("FavoriteListCell does not exist")
        }`